packages/web-components/src/components/va-privacy-agreement/va-privacy-agreement.tsx

```diff
@@ -64,27 +67,59 @@ export class VaPrivacyAgreement {
   }
 
   render() {
-    return (
-      <Host>
-        <va-checkbox
-          required
-          error={this.errorMessage()}
-          id="checkbox"
-          label="I have read and accept the privacy policy."
-          checked={this.checked}
-          onVaChange={this.handleCheckboxChange}
-        >
-          <span slot="description">
-            Please read and accept the&nbsp;
-            <a href="/privacy-policy/" target="_blank">
-              privacy policy 
-              <i class="fa-arrow-up-right-from-square" aria-hidden="true" role="img"></i>
-              <span class="sr-only">opens in a new window</span>
-            </a>.
-          </span>
-        </va-checkbox>
-      </Host>
-    );
-  }
+    const { uswds } = this;
 
+    if (uswds) {
+      return (
+        <Host>
+          {/* <div style={{borderTop: '1px solid red'}}></div> */}
+          <va-checkbox
+            uswds={true}
+            required
+            error={this.errorMessage()}
+            id="checkbox"
+            label="I have read and accept the privacy policy."
+            checked={this.checked}
+            onVaChange={this.handleCheckboxChange}
+          >
+            <span class="privacy-policy" slot="description">
+              Please read and accept the&nbsp;
+              <a href="/privacy-policy/" target="_blank">
+                privacy policy
+                <i
+                  class="fa-arrow-up-right-from-square"
+                  aria-hidden="true"
+                  role="img"
+                ></i>
+                <span class="sr-only">opens in a new window</span>
+              </a>
+              .
+            </span>
+          </va-checkbox>
+        </Host>
+      );
+    } else {
```

Comment:
I know that we have done an `else` in previous v3 variation updates but we don't really need it because the `return` statement will not be called twice. If we remove the `else`, we can also maintain a clean version history (the carriage returns make it appear that all of this code has changed when it has not).